components/bl-autocomplete-component/src/options/option.js

```diff
@@ -0,0 +1,37 @@
+export const Option = props => {
+  const {
+    item,
+    setInputValue,
+    setAutocompleteValue,
+    setIsOptionsOpen,
+    onAutocompleteChange,
+  } = props;
+
+  const handleClick = () => {
+    setInputValue('');
+    setAutocompleteValue(item);
+    setIsOptionsOpen(false);
+
+    if (onAutocompleteChange) {
+      onAutocompleteChange({ autocompleteValue: item });
+    }
+  };
+
+  const onEnterClick = ({ key }) => {
+    if (key === 'Enter') {
```

Comment:
Could we navigate between options using tab/arrows?

Reply:
We can navigate between options using a tab.